src/ripple/protocol/impl/STLedgerEntry.cpp

```diff
@@ -78,7 +78,7 @@ void STLedgerEntry::setSLEType ()
     type_ = mFormat->getType ();
     if (!setType (mFormat->elements))
     {
-        if (auto j = debugLog())
+        if (auto j = debugLog().error())
```

Comment:
What happens to a `Stream` if the `Journal` that produced it goes away? I _think_ that's safe - currently anyways. But this could break in weird and unexpected ways if that ever changes.